src/einsteinpy/integrators/fantasy.py

```diff
@@ -0,0 +1,252 @@
+import warnings
```

Comment:
We have Licence issues here right?

Reply:
I suppose. Honestly, I don't know, if we do.

Reply:
It does seem like, GPL3 licensed code cannot be included in MIT licensed projects.